### ISSUE TITLE
Stylesheet directory

### ISSUE THREAD

**Issue by theCoderCat** [Templarian/MaterialDesign#126]
Hi, I ran into this set of icons yesterday and then completely fall for it :)

But I think it would be nice to put the stylesheet into a directory which is sibling of fonts directory like below:

```
\
|_ fonts
|  |_MaterialDesignIcons.eot
|  |_MaterialDesignIcons.tff
|  |_MaterialDesignIcons.woff
|_styles
   |_styles.css
```
This way, MDI can work nicely along side with other icon-font sets (fontawesome, glyphicon-halfling,...)

Best regard :)

**Comment by Templarian** [comment]
Hmm... that might be a good idea. Since the font is still in alpha I don't mind changing it to `../fonts/`.

I do font builds every Sunday after reviewing the various icons that have been published that week, so it won't go in until then.

**Comment by theCoderCat** [comment]
Thank you for responding :) I'm looking forward to the next update :+1:

**Comment by Templarian** [comment]
It didn't make it into last weeks update. I'm coding up everything today. It will match:

```
/
| css
   | materialdeisgnicons.css
   | materialdesignicons.min.css
| fonts
   | MaterialDesignIcons.eot
   | MaterialDesignIcons.tff
   | MaterialDesignIcons.woff
| less
   | _icons.scss
   | _path.css
   | materialdesignicons.css
| scss
   | icons.less
   | path.less
   | materialdesignicons.less
| preview.html
```

There are a few open source projects that use the icons right now, so if I'm breaking it I'm making sure I break it good hence adding all these other files etc.

**Comment by theCoderCat** [comment]
Awesome :+1:

**Comment by Templarian** [comment]
## Update

I wrote the SCSS files last night. Everything is good on that side of things. Working on adding SVG / woff2 support to the build (SVG should be easy I had disabled it earlier due to some minor issues that should be resolved now that the font generation was recoded a while back). The SCSS will build the various CSS files.

## Note

I may release without the LESS files for this next update (I don't use LESS, so I need to read the documentation and make it look similar to the SCSS).

**Comment by theCoderCat** [comment]
Sound good, if someone ask for LESS version, may be I can help out later since I can write both SCSS and CSS :)

**Comment by Templarian** [comment]
This has been closed in `v0.9.21-alpha` (along with tons of other cool things). LESS is coming in the next release. I only got the SASS support in.

Thanks for opening this issue! :smile:
